.circleci/config.yml

```diff
@@ -1,79 +1,65 @@
 version: 2.1
 
-jobs:
-  lint_and_build:
-    docker:
-      - image: circleci/node:11.15
+commands:
+  install_dependencies:
+    description: "Install dependencies and save cache"
+    parameters:
+      key:
+        type: integer
     steps:
-      - checkout
       - restore_cache:
           name: Restore Yarn Package Cache
           keys:
-            - yarn-packages-waffle-11-{{ checksum "yarn.lock" }}
+            - yarn-packages-waffle-<< parameters.key >>-{{ checksum "yarn.lock" }}
       - run:
           name: Install dependencies
           command: yarn install --frozen-lockfile
       - save_cache:
           name: Save Yarn Package Cache
-          key: yarn-packages-waffle-11-{{ checksum "yarn.lock" }}
+          key: yarn-packages-waffle-<< parameters.key >>-{{ checksum "yarn.lock" }}
           paths:
             - ~/.cache/yarn
+  setup_solc:
+    steps:
       - run:
-          name: Lint
-          command: yarn lint
-      - run:
-          name: Build
-          command: yarn build
+          name: Setup native solc
+          command: sudo cp ~/project/.circleci/solc /bin
+
+
+jobs:
+  lint_and_build:
+    docker:
+      - image: circleci/node:11.15
+    steps:
+      - checkout
+      - install_dependencies:
+          key: 11
+      - run: yarn lint
+      - run: yarn build
```

Comment:
We are already building in with node 11 with `test_node_11` job, why duplicate?

Reply:
Will change, this will also change the name of the the job to `lint` from `lint_and_build` so you will need to change repo configuration to merge it